includes/payment-tokens/class-wc-payment-token-cc.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+/**
+ * WooCommerce Credit Card Payment Token
+ *
+ * Representation of a payment token for credit cards
+ * Extends from WC_Payment_Token_eCheck since both types contain last4 digits.
+ *
+ * @class 		WC_Payment_Token_CC
+ * @since		2.6.0
+ * @category 	PaymentTokens
+ * @package 	WooCommerce/PaymentTokens
+ * @author		WooThemes
+ */
+class WC_Payment_Token_CC extends WC_Payment_Token_eCheck {
```

Comment:
I would avoid having the CC token class extend the eCheck token class, for two reasons:

1) they're fundamentally different payment methods
2) core may need to introduce methods in the eCheck class that would conflict with methods in the CC and cause issues in the future